### ISSUE TITLE
[Enhancement] Data Table - Draggable rows 

### ISSUE THREAD

**Issue by ghost** [kirbydesign/designsystem#2526]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
Having draggable rows in the Kirby Data Table

## Describe the solution you'd like
Adding a flag to e.g. the <tbody kirby-tbody [draggable]=draggable>

## Have you considered any alternatives?
Creating the draggable logic locally and using the styled Data Table.

## Are there any additional context?
There are different ways of implementing, could be with an assigned column/cell in the given row that is the handle or the whole row is draggable.

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by RasmusKjeldgaard** [comment]
The Data Table is focused on display of data, not manipulation of data. It is closely coupled to the HTML Table and is only intended as a styled layer on top of what is already supported by the builtin table. 

A lot of excellent libraries like AG grid or Grid.js exist for advanced table use-cases, and we can definitely explore how tables made with external libraries can get Kirbys styles, but please open a separate issue if that is needed.
